tasks.rb

```diff
@@ -818,6 +820,17 @@ def set_hpxml_walls(hpxml_file, hpxml)
   elsif ['base-atticroof-cathedral.xml'].include? hpxml_file
     hpxml.walls[1].interior_adjacent_to = HPXML::LocationLivingSpace
     hpxml.walls[1].insulation_assembly_r_value = 23.0
+  elsif ['base-enclosure-tiny-wall.xml'].include? hpxml_file
+    hpxml.walls.add(id: 'TinyWall',
+                    exterior_adjacent_to: HPXML::LocationOutside,
+                    interior_adjacent_to: HPXML::LocationLivingSpace,
+                    wall_type: HPXML::WallTypeWoodStud,
+                    area: 0.05,
+                    solar_absorptance: 0.7,
+                    emittance: 0.92,
+                    insulation_assembly_r_value: 20.0)
```

Comment:
This adds another simulation for a fairly uncommon problem. It also only checks walls.

I suggest 1) Removing this test file and adding this surface to the base-enclosure-split-surfaces.xml file instead, and 2) Adding similar tiny surface for roofs, rim joists, foundation walls, windows, etc.